packages/paste-core/components/form/src/FormInput.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import css from '@styled-system/css';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+
+export type InputTypes = 'text' | 'email' | 'hidden' | 'number' | 'password' | 'search' | 'tel';
+
+export interface FormInputProps extends React.InputHTMLAttributes<''> {
+  id: string;
+  type: InputTypes;
+  name?: string;
+  value?: string;
+  placeholder?: string;
+  disabled?: boolean;
+  readOnly?: boolean;
+  required?: boolean;
+  hasError?: boolean;
+  className?: never;
+  style?: never;
+  size?: never;
+  height?: never;
+  width?: never;
+}
+
+// eslint-disable-next-line emotion/syntax-preference
+const StyledFormInput = styled(Box)<FormInputProps>(({hasError}: FormInputProps) =>
+  css({
+    appearance: 'none',
+    outline: 'none',
+    resize: 'none',
+    width: '100%',
+    backgroundColor: 'colorBackgroundBody',
+    borderWidth: 'borderWidth10',
```

Comment:
Require here is that input height matches the button height. So we're going to have to remove that border and use a box-shadow to get the math to line up.

Reply:
I've setup new shadow tokens and replaced all of the borders with the box-shadows.